### PR TITLE
Bighorn Power Plant (Properly, this time)

### PR DESCRIPTION
**Pull request by ValiantEsp**
Cleaned up the commits, used Map Merge 2 to actually merge the map properly. Adds a power plant to Bighorn, adds a construction area, sets Bighorn, the Followers, and Bighorn Underground to require power, adds access to Followers and Bighorners, puts the Schoolhouse on the Follower Grid, changes the Schoolhouse to match Follower construction.

Take two on the power-plant PR. This time done far more cleanly. Thank you, Carl!

I'll be honest, it's more a step in re-adding engineering as a whole. It means the town will require power to keep the lights on, and be able to produce it instead of relying on a magical infinite source. May have more meaning as more of the Wasteland is given it's own power-grid. Baby-steps, though! It's also my first foray into proper mapping.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog

:cl:
add: Added a Powerplant, keyed to EngineTools access, with Substations keyed to Follower and Bar (Town) access.
add: Added a Construction Site, keeping the amount of 'trash' stable while the map changes are done.
change: Replaces the Nightstalker Den near Bighorn.
change: Set Bighorn, the Followers, and Bighorn Underground to require power.
change: Cleans up the Followers Schoolhouse, since it's been set on the Follower Grid.
change: Bighorn Underground is now it's own area, separate from the sewers.
/:cl:

### CODE REVIEW

File: code/modules/jobs/job_types/bighorn.dm [hunk 1]
```diff
@@ -434,8 +434,8 @@ Mayor
 	/datum/outfit/loadout/cleanser		//Just some bombs.
 	)
 
-	access = list()		//we can expand on this and make alterations as people suggest different loadouts
-	minimal_access = list()
+	access = list(ACCESS_BAR)		//we can expand on this and make alterations as people suggest different loadouts
```
Comment (ProbablyCarl) [inline code/modules/jobs/job_types/bighorn.dm:437]: Preacher becoming an alcoholic, yo????

**Comment by ProbablyCarl** [comment]
What's up with the extended follower walkway interior?

**Comment by ProbablyCarl** [comment]
What's this?
image

File: code/modules/jobs/job_types/bighorn.dm (hunk 1 above)
Comment (ValiantEsp) [reply code/modules/jobs/job_types/bighorn.dm:437]: The Bighorn Bar's access is Kitchen.
The town's own access, for like the workshop/ect, is Bar.

**Comment by ValiantEsp** [comment]
> Substations to actually POWER the town, underground, exterior, and Followers. Until areas are messed with a lot more in Bighorn 2.0, this is where the APCs are for each area.

**Comment by ValiantEsp** [comment]
> What's up with the extended follower walkway interior?
I don't know what this is referring to.
Walkway

File: code/modules/jobs/job_types/bighorn.dm (hunk 1 above)
Comment (ValiantEsp) [reply code/modules/jobs/job_types/bighorn.dm:437]: I.. Honestly don't know why that is, but I figure actual town access would be something to fix for BH2.0

**Comment by ValiantEsp** [comment]
OH. I understand now. The upper walk-way? It's been extended to match the area below and look a little cleaner, is all, since the construction area and powerplant are bulging out the side.

**Comment by ProbablyCarl** [comment]
I'll merge this momentarily. Good stuff.
Looks good at a glance with the above taken into consideration.

**Pull request merged**